package.json

```diff
@@ -13,7 +13,7 @@
     "preline": "^1.9.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-infinite-scroll-component": "^6.1.0",
+    "react-intersection-observer": "^9.5.3",
```

Comment:
nice job finding this library and applying it this way!

Reply:
Found it in an react query example lol